### PR TITLE
fix(spanner): DeleteSampleData sample

### PR DESCRIPTION
**Pull request by laljikanjareeya**
[PR#1111](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/1111) failing [TestDeleteSampleData](https://source.cloud.google.com/results/invocations/873b1a6a-39fd-4d49-b254-817029ee8a3a/targets/github%2Fdotnet-docs-samples%2Fspanner%2Fapi%2FSpannerTest%2FTestResults/tests) Test as in sample was not added `await` while deleting the tables.
It's giving below error:
`Error: Google.Cloud.Spanner.Data.SpannerException : Cannot drop table UpcomingSingers with interleaved tables: UpcomingAlbums.`

### CODE REVIEW

**Comment by laljikanjareeya** [comment]
@googlebot I fixed it.

File: spanner/api/Spanner/Program.cs [hunk 1]
```diff
@@ -2016,11 +2016,11 @@ AlbumTitle   STRING(MAX)
...
             using (var connection = new SpannerConnection(connectionString))
             {
-                DeleteTable(connection, "UpcomingAlbums");
-                DeleteTable(connection, "UpcomingSingers");
+                await DeleteTable(connection, "UpcomingAlbums");
+                await DeleteTable(connection, "UpcomingSingers");
                 return 0;
             }
-            async void DeleteTable(SpannerConnection connection, string table)
+            async Task DeleteTable(SpannerConnection connection, string table)
```
Comment (amanda-tarafa) [inline spanner/api/Spanner/Program.cs:2023]: nit: rename to DeleteTableAsync

**Comment by laljikanjareeya** [comment]
I fixed it.

*Thanks & Regards*
*Lalji Kanjareeya*

**Comment by skuruppu** [comment]
@amanda-tarafa the last test run has an unrelated [failure](https://source.cloud.google.com/results/invocations/5cd50161-0286-4fe9-9a72-5bf063d2b0f6/targets). So we'll need your help to force-merge this once the new set of runs finish assuming there are no spanner test failures.

**Pull request merged**